ga.rpmtw.rpmlauncher.metainfo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>ga.rpmtw.rpmlauncher</id>
+
+    <name>RPMLauncher</name>
+    <summary>A multi-functional Minecraft Launcher power by The RPMTW Team,Made with Flutter and Dart.</summary>
+    <developer_name>siongsng</developer_name>
+
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>GPL-3.0</project_license>
+
+    <url type="homepage">https://github.com/RPMTW/RPMLauncher</url>
+
+    <description>
+        <p>
+            A multi-functional Minecraft Launcher power by The RPMTW Team,Made with Flutter and Dart.
```

Comment:
```suggestion
            A multi-functional Minecraft Launcher power by The RPMTW Team.
```

It would be nice to have more info about how this is different from other launchers, for *users*